split/groupresource.go

```diff
@@ -0,0 +1,160 @@
+package split
+
+import (
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/go-openapi/spec"
+	"github.com/iancoleman/strcase"
+
+	"github.com/kubewarden/k8s-objects-generator/object_templates"
+	"github.com/kubewarden/k8s-objects-generator/swagger_helpers"
+)
+
+const (
+	kubernetesGroupVersionKindKey = "x-kubernetes-group-version-kind"
+)
+
+type groupVersionResource struct {
+	Group   string
+	Version string
+	Kind    string
+}
+
+func GenerateGroupResources(project Project, plan *RefactoringPlan) error {
+	objectKindTemplate, err := template.New("objectKind").Parse(object_templates.ObjectKindTemplate)
+	if err != nil {
+		return err
+	}
+	groupVersionTemplate, err := template.New("groupVersion").Parse(object_templates.GroupVersionTemplate)
+	if err != nil {
+		return err
```

Comment:
Can you add some context to the error to make it easier to understand which template caused the error?

Reply:
The same as above